packages/rancher-istio-1.9/package.yaml

```diff
@@ -1,2 +1,2 @@
 url: local
-packageVersion: 0
+packageVersion: 1
```

Comment:
Needs update here: https://github.com/rancher/rancher/issues/33528

Reply:
I have updated the issue - it now includes the OOB release info, and this new release info.